### DISCUSSION TITLE
How to dump the private keys from a specific adress?

### DISCUSSION THREAD

**Discussion by cideight** [ordinals/ord#1955] [Q&A]
Hi 
Use only ord wallet for make inscriptions. How i can dump my private keys for the existing adresses?
I got the MNEMONICS  and the wallet.dat.  
What is the safest way to secure the inscriptions outside the computer network.

i tried with:
bitcoin-cli dumpp2wpkhkey "bc1....adress that holds the inscription" = error code: -32601 error message: Method not found
bitcoin-cli getaddressinfo "bc1...." = error code: -5 error message: Invalid checksum
bitcoin-cli dumpprivkey "bc1.... > error code: -4 error message: Only legacy wallets are supported by this command

Thank for helping me out.

**Comment by nammaki** [comment]
It seems intentionally unsupported according to [this](https://bitcoin.stackexchange.com/questions/107216/how-do-i-get-the-private-key-of-an-address-in-descriptor-wallets-how-to-dumppri) ❤️

**Comment by eskabe** [comment]
https://github.com/Anderson-Juhasc/bip86
